tests/testthat/test-indention_operators.R

```diff
@@ -47,12 +61,16 @@ test_that(
         "indention_operators", "not_first_trigger",
         transformer = style_text
       ),
-    NA)
-})
+      NA
+    )
+  }
+)
```

Comment:
that looks strange.

Reply:
@krlmlr would you agree that there should be no line break if the token before `)` is `}`?

Reply:
I think that would comply with [section 2.4](http://style.tidyverse.org/syntax.html#spacing), although function calls are not explicitly mentioned there.

Reply:
Fixed that in some branch, is a minor thing.

Reply:
This looks correct, except the `{` should be on its own line.

Reply:
Reference: https://github.com/krlmlr/styler/issues/125#issuecomment-323018778.